spec/tracking_spec.js

```diff
@@ -0,0 +1,92 @@
+describe('App Tracking', function() {
+  var tracking = require('tracking'),
+      sandbox  = sinon.sandbox.create(),
```

Comment:
Where does `sinon` come from?

Reply:
[Sinon](http://sinonjs.org/) is the stubs/mocks/spies library this uses.

Reply:
And it's available without needing `require`? Odd.